library/analytics/src/debug/AndroidManifest.xml

```diff
@@ -0,0 +1,26 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools">
+
+    <application>
+        <!--suppress AndroidDomInspection -->
+        <activity
+            android:name="org.keynote.godtools.android.activity.MainActivity"
+            tools:ignore="MissingClass">
+            <!-- AppsFlyer deep links -->
+            <intent-filter android:autoVerify="true">
+                <action android:name="android.intent.action.VIEW" />
+
+                <category android:name="android.intent.category.DEFAULT" />
+                <category android:name="android.intent.category.BROWSABLE" />
+
+                <data android:scheme="http" />
+                <data android:scheme="https" />
+                <data android:host="godtools.onelink.me" />
```

Comment:
Does AppsFlyer provide like a default AndroidManifest that we then add our values to?

Reply:
AppsFlyer itself doesn't, we are adding these values to the `MainActivity` that is defined in the `app` module. there is an AndroidManifest merge process that combines the manifest from all included libraries to generate the final manifest.

the reason I put the filters in the `analytics` module instead of directly in the `app` module was to try and keep all AppsFlyer related logic within the `analytics` module if possible.